### ISSUE TITLE
Bug: Class declarations are not supported in Functionless AST

### ISSUE THREAD

**Issue by thantos** [functionless/functionless#327]
```ts
const func = new Function<undefined, void>(stack, "func", async () => {
  class MyError extends Error {
    constructor(readonly value: string, message: string) {
      super(message);
    }
  }
  
  throw new MyError("val", "wat");
});
```

```console
Error: unhandled node: class MyError extends Error {
    constructor(readonly value: string, message: string) {
      super(message);
    }
  } ClassDeclaration
```

**Comment by sam-goodwin** [comment]
More reason to migrate to TS AST?

**Comment by thantos** [comment]
Fixed by #347
